tests/containers/bci_version_check.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Checks if the container version for the test run is still up-to-date
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use utils qw(zypper_call script_retry);
+use db_utils qw(push_image_data_to_db);
+use containers::common;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    select_serial_terminal;
```

Comment:
This above also is in the `bci_prepare.pm`: could it be removed from here?

Reply:
No, every test should ensure that the proper terminal is set.

Otherwise if you would change the module in `bci_prepare`, you would need to know which other test runs depend on it for doing so.